### DISCUSSION TITLE
[AXE] Price is back to 1 SFL

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1528] [General]
That was the unique advantage for players to level up and now you change it without a warning.

It also affects Iron and Gold price. If I had been warned, I would change the price of my listed items or change my mind to sell my items.

Do you think that is it correct to change it without a warning?

This kind of decision seems like you guys don't care about it affects.

**Comment by future2back** [comment]
Always call attention to the information clear within the game and now i realized that the value has changed after i saw that my money ran out.


Please take care of your players.

**Reply by Myth727** [reply]
I think Adam is doing a great job, a few tiny things happen now and then. Would you be down for me sending you some wood 🪵? To make up for the lack of announcement? I enjoy seeing members of our community looking for issues and talking about them :)

I got 250 wood set aside for you if you'd like.

**Reply by On1el** [reply]
Where is the best community - in our club)

**Reply by future2back** [reply]
@Myth727 Thank you for participating, it's not the first time you've appeared in discussions I've opened. To be clear, the main idea here is not to for a mitigation. Is to bring clearer announcements in game, mainly due to this importance. I could have prepared myself better or at least thought on it for my goal of reaching lvl 5 on Tools and get Apprentice Beaver, not sure if they will be reacheable.

Anyway if you want to contribute, here my farm address:

_0xeC8AD40119eFbe745eB6f93DBca0D8F154319FF9_

Thank you for your participation.

**Reply by Myth727** [reply]
Sent the 250 wood over your way @future2back 

https://polygonscan.com/tx/0xc55aa095ab7e6a61026d8ab39788494f50b2739323c427f4fc569229f801c91e

**Reply by future2back** [reply]
You crazy man @Myth727 this resource will help me lot. Thank you.

**Reply by Myth727** [reply]
No worries,  I also gifted you some eggs, wheat and stones. Took a peak at your farm and figured I'd help just a tad.

**Reply by future2back** [reply]
No words to thank you. This resources Unburdened me. Success brother 🚀 .

**Comment by adamhannigan** [comment]
That is a fair call @future2back 

When we announced the price drop, it was delivered as a 'temporary' drop.

The initial discussions we had with the community was that it would return to normal after 1-2 weeks. Unfortunately due to team business this was delayed and took roughly 3 weeks. The upside is that players got an extra week with the cheap axe prices.

You are right, this should of been communicated at time of announcement with in game modals.

**Reply by future2back** [reply]
Thanks @adamhannigan !

I don't remember exactly where I saw the announcement of axe price was down, I believe it was on Discord. I didn't know it was for a limited time or didn't pay attention to be honest, sorry. What I'd like to call attention here are the in-game announcements. In this case, i believe, price down and price up should be announced in-game due to the importance of the information. 

How many of the latest announcements are we seeing in-game? Could we have a link to sunflower-land official page with a complete history of all announcements made in-game?

Will we have more limited-time offers? I believe using timer on this could be more intuitive. 

Just to illustrate the idea:

![image](https://user-images.githubusercontent.com/74167741/197122767-408b8833-ee8b-4b71-9055-00ebe4148072.png)


Thank you.

**Reply by adamhannigan** [reply]
Agreed @future2back 

The comms in Beta have not been great and is something we are focussing on improving with land expansion 🙏
